actions/deploy-renku/entrypoint.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+RENKU_NAMESPACE=${RENKU_NAMESPACE:-$RENKU_RELEASE}
+RENKU_TMP_NAMESPACE=${RENKU_TMP_NAMESPACE:-${RENKU_NAMESPACE}-tmp}
```

Comment:
This is very convenient!
Should it be extended to the `teardown-renku` action as well?

Reply:
good point, thanks!